src/api/public/apidocs-new/paths/source_project_name_cmd_modifychannels.yaml

```
@@ -0,0 +1,55 @@
+post:
+  summary: Modify existing channels.
+  description: Add repositories or enable them for existing channels.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: query
+      name: mode
+      schema:
+        type: string
+        enum: [add_disabled, enable_all]
+      example: add_disabled
+      description: The channel will be added to the package using this mode.
+  responses:
+    '200':
+      description: ok
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          example:
+            code: ok
+            summary: Ok
```

```suggestion
    '200':
      $ref: '../components/responses/succeeded.yaml'
```
Nitpick: we could use the component instead.

---

Applied :+1: